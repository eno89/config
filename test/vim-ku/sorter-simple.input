" Test: vim-ku sorter-simple
function s:describe__ku_sorter_simple_sort()  "{{{1
  let args = {}
  let candidates = [
  \     {'word': 'j'},
  \     {'word': 'u'},
  \     {'word': 'n'},
  \     {'word': 'i'},
  \   ]

  It sort candidates by dictionary order

  Should ku#sorter#simple#sort(copy(candidates), args)
  \ ==# [{'word': 'i'}, {'word': 'j'}, {'word': 'n'}, {'word': 'u'}]
endfunction




" __END__  "{{{1
" vim: filetype=vim foldmethod=marker
