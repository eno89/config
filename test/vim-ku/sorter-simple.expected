==== ku_sorter_simple_sort 
---- It sort candidates by dictionary order 
.



**** Result ****


1 examples, 0 failures
